### ISSUE TITLE
We need a doc explaining that we don't have the data on policy docs before certain dates 

### ISSUE THREAD

**Issue by aoifespenge** [wellcometrust/policytool#103]
This concern arose from working on the internal policy dashboards. If we don't have the scraped data prior to a certain date we need to have a note about this on the dashboard. This is so that people who are using the dashboard don't just see the dates and assume that there were no policy docs before a certain date. While I know this sounds unlikely, at very least we will leave them feeling a bit uncertain about the data. This is particularly important when comparing between when the research was published and when it was cited.  Would it be possible to have a note of this somewhere in the policytool repo, from which I can copy into a Tableau dashboard note? Tagging everyone this is relevant to. 

<img width="385" alt="Screenshot 2019-03-13 at 09 58 39" src="https://user-images.githubusercontent.com/25910370/54271698-f5e07c80-4579-11e9-8ad5-3395441d2ce4.png">

<img width="372" alt="Screenshot 2019-03-13 at 10 13 08" src="https://user-images.githubusercontent.com/25910370/54271704-f9740380-4579-11e9-8269-025f27dd4635.png">

**Comment by nsorros** [comment]
Kudos for opening up an issue. I think @SamDepardieu is in the best position to answer this but as far as I am aware, the dates that documents are available pretty much depend on the organisation so it seems more like a dashboard design issue than a software engineering one.

**Comment by SamDepardieu** [comment]
Yup, we had a chat about it this morning. I agree that it's a good idea.
Here's a sum up of what I answered to this issue this morning, to give you a track of what need to be done in order to get these stats:
```
It’s a bit tricky, because not all orgs have the same starting year for their docs or an option to choose the starting year:
 * I can say for WHO (it's in the settings file)
 * I can look into the database for Nice
 * Gov should also be in the DB or in the settings
 * We’re not scraping the publication date for unicef/parliament/msf, if I remember well, so the only way to know it is to check on their websites the year of their first publication
```

**Comment by aoifespenge** [comment]
@nsorros we're quite limited by Tableau so the only way of informing the user of what dates are missing is a note about it. Otherwise, we might have something like a greyed out area. 

Just a few lines explaining the dates we start scraping the data for different orgs would be great.
